Domain/Entities/BetQuotes.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Domain.Entities
+{
+    public class BetQuotes
+    {
+        public Guid Id { get; private set; }
+
+        public Guid BetId { get; set; }
+
+        [Column(TypeName = "decimal(5, 2)")]
```

Comment:
this attribute is not needed since you are specifying the type in the db context